packages/labs/ssr/src/lib/render-lit-html.ts

```diff
@@ -44,19 +44,15 @@ import {
 
 import {escapeHtml} from './util/escape-html.js';
 
-import {
-  traverse,
-  parseFragment,
-  isCommentNode,
-  isElement,
-} from './util/parse5-utils.js';
+import {parseFragment} from 'parse5';
+import {isElementNode, isCommentNode, traverse} from '@parse5/tools';
```

Comment:
Is there a reason to use these over the TreeAdapter methods?

Reply:
It's just for simplicity really. So you can import the full set of helpers from one place. 

The tree adapter doesn't have every type guard function. In the utils package, I re export the ones it has _plus_ the extras it doesn't have. So you get them from a single place rather than two